src/main/java/ua/javarush/island/gameisland/OrganismFactory.java

```diff
@@ -0,0 +1,90 @@
+package ua.javarush.island.gameisland;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import org.apache.commons.lang3.RandomStringUtils;
+import ua.javarush.island.entity.Organism;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class OrganismFactory {
+
+    public static final  String PATH_TO_ORGANISM_FOLDER = "ua.javarush.island.entity.organism.";
+    public static final  String PATH_TO_YAML_FOLDER = "./src/main/resources/";
+
+    public static void createOrganism(Map<String, Integer> residentsProperties) {
+
+        for (String name : residentsProperties.keySet()) {
+
+            String fullNameClass = PATH_TO_ORGANISM_FOLDER + name;
+            int quantity = residentsProperties.get(name);
+
+            if (quantity >0) {
+                try {
+                    Path pathToYamlConfigFile = Path.of( PATH_TO_YAML_FOLDER+ name + ".yaml");
+                    if (!Files.exists(pathToYamlConfigFile)) {
+                        System.out.println("File " + name + ".YAML not exist. Object not create ");
+                    }
+
+                    Class<?> organismClass = Class.forName(fullNameClass);
+
+                    ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
```

Comment:
ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
                    mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
                    
                    Can be configure once, no need to create an instance of objectmapper for each method invocation